HW_15/Motoshop/Motoshop/Attributes/MinYearAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Motoshop.Models;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Motoshop.Attributes
+{
+    public class MinYearAttribute : ValidationAttribute
+    {
+        public int MinYear { get; }
+        public MinYearAttribute(int minYear)
+        {
+            MinYear = minYear;
+        }
+        public string GetErrorMessage() => $"Motorcycle must be made no early than {MinYear}";
```

Comment:
replace to 
![image](https://user-images.githubusercontent.com/3023032/129870718-15bc7488-8463-403e-bfeb-432e3509d58b.png)